components/listingsPage/results.js

```diff
@@ -52,7 +52,8 @@ const Results = ({ t }) => {
           width: "100%",
         }}
       >
-        Showing 1 -10 of 1000 firms that match
+        {t("headings.showing")} 1 -10 {t("headings.of")} 1000{" "}
+        {t("headings.firms")}
```

Comment:
I am wondering if there is a more elegant way of rendering this copy. Is it possible for instance to pass the numbers into the whole sentence as variables? Otherwise this is fine but I can see a situation where the potentially differing word orders across languages might cause problems.

Also 
- can we be a little more consistent around dashes (`1 -10`)?
- is it necessary to explicitly include a space(`{" "}`)?

Reply:
There is. Just considering the fact that some kind of logic will be rendering the figures and don't want to go ahead of myself. About the space, thats the linter that added that. I am removing it now.

Reply:
OK, thanks, all good! 👍

Reply:
That kind of logic can definitely be done using `i18next` but that can be done in a later date yeah